reactive/docker_registry.py

```diff
@@ -246,13 +246,20 @@ def update_reverseproxy_config():
     #  https://bugs.launchpad.net/layer-docker-registry/+bug/1815459
     common_opts = "check inter 2000 rise 2 fall 5 maxconn 4096"
     is_primary = True
+    tls_ops = ""
```

Comment:
Super knit-picky, but can you call this `tls_opts` vs `_ops`?  That's consistent with `common_opts` and `server_opts` in this function.

Reply:
whoops, shoulda spotted that one